mod_reforged/hooks/skills/effects/taunted_effect.nut

```diff
@@ -1,13 +1,15 @@
 ::Reforged.HooksMod.hook("scripts/skills/effects/taunted_effect", function(q) {
+	q.m.DefenseModifier <- 0;
+
 	q.create = @(__original) function()
 	{
 		__original();
-        this.m.Description = "This character is taunted by another character and is much more likely to engage and attack them.";
+		this.m.Description = "This character is taunted by another character and is much more likely to engage and attack them.";
```

Comment:
For cleanliness I'd remove this whitespace change from this commit. It makes going through commit history later especially with "blame file" more polluted. We should do a full codebase conversion of spaces to tabs in a single commit later.

Reply:
Note that I put this specific change in a commit named `refactor(style): improve formatting of taunted_effect` because there were a few other things that bugged me about that class. In this context I find it reasonable to replace whitespaces.